quic/s2n-quic-core/src/recovery/bbr/full_pipe.rs

```diff
@@ -136,6 +142,33 @@ impl Estimator {
         false
     }
 
+    /// Determines if enough consecutive rounds of explicit congestion have been encountered that we
+    /// can estimate the available bandwidth has been fully utilized.
+    ///
+    /// Based on bbr2_check_ecn_too_high_in_startup from https://github.com/google/bbr/blob/1a45fd4faf30229a3d3116de7bfe9d2f933d3562/net/ipv4/tcp_bbr2.c#L1372
+    fn excessive_explicit_congestion(
+        &mut self,
+        rate_sample: bandwidth::RateSample,
+        max_datagram_size: u16,
+    ) -> bool {
+        // Startup is exited if the number of consecutive round trips with ECN CE markings above
+        // the ECN_THRESH exceed this value
+        // Value from https://github.com/google/bbr/blob/1a45fd4faf30229a3d3116de7bfe9d2f933d3562/net/ipv4/tcp_bbr2.c#L2334
+        const STARTUP_FULL_ECN_COUNT: u8 = 2;
```

Comment:
does it make more sense to declare this at the top-of-file instead of in the func?

Reply:
My rule of thumb has generally been to include it at the top if its used in multiple places or is generally a fundamental constant important to the whole file. In this case, I only use it in this one place, so I put it closest to where its used

Reply:
I agree with the approach and have done the same.